src/utils/queries/clientQuery.js

```diff
@@ -0,0 +1,14 @@
+import ApolloClient from 'apollo-boost';
+
+const client = new ApolloClient({
+  uri: 'https://cors-anywhere.herokuapp.com/https://pure-castle-14648.herokuapp.com/graphql'
+})
+
+// This is the function that "posts" the query, the client automatically sends the query object from the function argument.
+// A failed query also triggers the catch method
```

Comment:
Cool! Can we walk through the client together this afternoon? Is this where we would make all of our custom queries?

Reply:
Upvote!